index.js

```diff
@@ -1,37 +1,61 @@
-// A convenience file to allow you to `require` all of the components defined in this module.
+// The main file that is included when you run `require("gpii-express")`.
+//
+// You can optionally load testing support by assigning a variable to the results of `require` and calling
+// `loadTestingSupport`, as in:
+//
+//      var gpiiExpress = require("gpii-express");
+//      gpiiExpress.loadTestingSupport();
+//
+"use strict";
+var fluid = require("infusion");
+var express = fluid.registerNamespace("gpii.express");
+
+var loader = fluid.getLoader(__dirname);
```

Comment:
This would be the appropriate place to write the line
fluid.module.register("gpii-express", __dirname, require) 

that advertises the base directory of this module for use in variable interpolation